src/xcode/ENA/ENA/Resources/Localization/en.lproj/usage.html

```diff
@@ -174,45 +174,19 @@
 		<h2>
 			CONTENTS
 		</h2>
-		<p>
-			1...... What features does the App have?
-		</p>
-		<p>
-			2...... What does low or elevated risk mean?
-		</p>
-		<p>
-				3...... What should I do if I have been exposed to a confirmed case?
-		</p>
-		<p>
-				4...... The App does not protect you against infection
-		</p>
-		<p>
-			5...... Retrieval of test results
-		</p>
-		<p>
-			6...... Activating the warning feature
-		</p>
-		<p>
-			7...... Requirements for using the App
-		</p>
-		<p>
-			8...... Limitations of the App
-		</p>
-		<p>
-			9...... Rights of use
-		</p>
-		<p>
-				10.... Important information about availability and changes
-		</p>
-		<p>
-			11.... No warranty
-		</p>
-		<p>
-				12.... Special terms for the iOS version of the App
-		</p>
-		<p>
-			13.... Final provisions
-		</p>
+		<p>1. What features does the App have?</p>
+		<p>2. What does low or elevated risk mean?</p>
+		<p>3. What should I do if I have been exposed to a confirmed case?</p>
+		<p>4. The App does not protect you against infection</p>
+		<p>5. Retrieval of test results</p>
+		<p>6. Activating the warning feature</p>
+		<p>7. Requirements for using the App</p>
+		<p>8. Limitations of the App</p>
+		<p>9. Rights of use</p>
+		<p>10. Important information about availability and changes</p>
+		<p>11. No warranty</p>
+		<p>12. Special terms for the iOS version of the App</p>
+		<p>13. Final provisions</p>

```

Comment:
We really need a sophisticated solution here. Not only will, for example an `<ol>`, create cleaner semantics but also help with accessibility for screen readers!

Reply:
Converting the MSWord document to html is a pain. There is still manual formatting required, and the CSS styles  are not working reliably.